### PR TITLE
Create fr translation of some parts of migrations folders

### PR DESCRIPTION
**Pull request by happytodev**
Hello, I had already translated some files before but under an another account 'itanea’.

Now, I work with this account 'happytodev'.

I don't know if it is possible to merge the two accounts, if not I will continue to make PR for the fr translation.

Don't hesitate to tell me if something is wrong.

Thank you.

### CODE REVIEW

File: appendices/migration82/windows-support.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ <title>Support de Windows</title>
+
+ <sect2 xml:id="migration82.windows-support.core">
+  <title>Core</title>
+  <para>
+   Les messages d'erreur spécifiques à Windows ne sont plus traduits, à la place ils sont
+   toujours affichés en anglais pour mieux correspondre aux messages d'erreur de PHP.
+  </para>
+  <para>
+   Un support préliminaire et hautement expérimental pour la construction sur ARM64 a été ajouté.
```
Comment (devnexen) [inline appendices/migration82/windows-support.xml:14]: I would suggest `la compilation pour (or sous) ARM64` instead.

File: appendices/migration82/windows-support.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  </para>
+ </sect2>
+
+ <sect2 xml:id="migration82.windows-support.oci8">
+  <title>OCI8</title>
+  <para>
+   Étant donné que la construction pour Oracle Client 10g n'est plus supportée de
```
Comment (devnexen) [inline appendices/migration82/windows-support.xml:21]: `Étant donné que compiler pour`

File: appendices/migration82/windows-support.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+   Les options de configurations <command>--with-oci8-11g</command>, <command>--with-oci8-12c</command> et
+   <command>--with-oci8-19</command> sont toujours prises en charge.
+  </para>
+ </sect2>
+
+ <sect2 xml:id="migration82.windows-support.zip">
+  <title>Zip</title>
+  <para>
+   L'extension Zip est mise à jour à la version 1.21.0, 
+   et est maintenant construite comme une bibliothèque partagée (<acronym>DLL</acronym>) par défaut.
```
Comment (devnexen) [inline appendices/migration82/windows-support.xml:32]: `est maintenant compilée comme une bibliothèque partagée`

File: appendices/migration81/deprecated.xml [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+   <title>Vérification des arguments autres que des chaînes de caractères</title>
+
+   <para>
+    Le passage d'un argument qui n'est pas une chaîne de caractères est déprécié.
+    Dans le futur, l'argument sera interprété comme une chaîne de caractères au lieu de
+    d'un point de code ASCII.
+    Selon le comportement souhaité, l'argument doit être soit
+    de caster en &string; soit de faire un appel explicite à
+    <fonction>chr</function>.
+    Toutes les fonctions <literal>ctype_*()</literal> fsont affectés.
```
Comment (adoy) [inline appendices/migration81/deprecated.xml:160]:
```suggestion
    Toutes les fonctions <literal>ctype_*()</literal> sont affectées.
```

File: appendices/migration81/deprecated.xml [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+
+ <sect2 xml:id="migration81.deprecated.hash">
+  <title>Hash</title>
+
+  <para>
+   <function>mhash</function>,
+   <function>mhash_keygen_s2k</function>,
+   <function>mhash_count</function>,
+   <function>mhash_get_block_size</function>,
+   et <function>mhash_get_hash_name</function> ont étés dépréciés.
```
Comment (adoy) [inline appendices/migration81/deprecated.xml:218]:
```suggestion
   et <function>mhash_get_hash_name</function> ont étés dépréciées.
```

File: appendices/migration81/deprecated.xml [hunk 6]
```diff
@@ -0,0 +1,369 @@
...
+  </para>
+ </sect2>
+
+ <sect2 xml:id="migration81.deprecated.mysqli">
+  <title>MySQLi</title>
+
+  <para>
+   La propriété <property>mysqli_driver::$driver_version</property> 
+   a été dépréciée.
+   Il n'avait pas de sens et était dépassé, utilisez <constant>PHP_VERSION_ID</constant>
```
Comment (adoy) [inline appendices/migration81/deprecated.xml:258]: À quoi fait référence `Il` ?

File: appendices/migration81/deprecated.xml [hunk 7]
```diff
@@ -0,0 +1,369 @@
...
+   à la place
+  </para>
+
+  <para>
+   L'appel de <methodname>mysqli::get_client_info</methodname> ou
+   <function>mysqli_get_client_info</function> avec l'argument
+   <parameter>mysqli</parameter> a été déprécié.
+   Appelez la <function>mysqli_get_client_info</function> sans aucun argument
```
Comment (adoy) [inline appendices/migration81/deprecated.xml:266]:
```suggestion
   Appelez <function>mysqli_get_client_info</function> sans aucun argument   
```

File: appendices/migration81/deprecated.xml [hunk 8]
```diff
@@ -0,0 +1,369 @@
...
+   L'appel de <methodname>mysqli::get_client_info</methodname> ou
+   <function>mysqli_get_client_info</function> avec l'argument
+   <parameter>mysqli</parameter> a été déprécié.
+   Appelez la <function>mysqli_get_client_info</function> sans aucun argument   
+   pour obtenir les informations sur la version de la bibliothèque client.
+  </para>
+
+  <para>
+   La méthode <methodname>mysqli::init</methodname> a été dépréciée.
+   Pemplacez les appels à <methodname>parent::init</methodname> par
```
Comment (adoy) [inline appendices/migration81/deprecated.xml:272]:
```suggestion
   Remplacez les appels à <methodname>parent::init</methodname> par
```

File: appendices/migration81/deprecated.xml [hunk 9]
```diff
@@ -0,0 +1,369 @@
...
+
+  <para>
+   La directive INI <link linkend="ini.auto-detect-line-endings">auto_detect_line_endings</link>
+   est dépréciée.
+   Si nécessaire, gérez plutôt les sauts de ligne <literal>"\r"</literal> manuellement.
+  </para>
+
+  <para>
+   Les constantes <constant>FILE_BINARY</constant> and
+   <constant>FILE_TEXT</constant> ont été déprécié.
```
Comment (adoy) [inline appendices/migration81/deprecated.xml:344]:
```suggestion
   <constant>FILE_TEXT</constant> ont été dépréciées.
```

**Comment by happytodev** [comment]
Thanks for all your recommendations. I will integrate them.
But as you know, I made some errors by pushing the master branch of my forks.
So, with help of @devnexen, I will try to recreate PRs from scratch.
If I succeed, I will close the 2 PR I submit previously.
I'm sorry for this pollution, but it's a new process for me.

**Pull request closed without merging**